packages/eas-cli/src/build/build.ts

```diff
@@ -649,3 +654,59 @@ function formatAccountSubscriptionsUrl(accountName: string): string {
     getExpoWebsiteBaseUrl()
   ).toString();
 }
+
+async function createAndUploadFingerprintAsync<T extends Platform>(
+  ctx: BuildContext<T>
+): Promise<{
+  runtimeVersion?: string;
+  fingerprintGCSBucketKey?: string;
+}> {
+  const resolvedRuntimeVersion = await resolveRuntimeVersionAsync({
+    exp: ctx.exp,
+    platform: ctx.platform,
+    workflow: ctx.workflow,
+    projectDir: ctx.projectDir,
+    env: ctx.buildProfile.env,
+    cwd: ctx.projectDir,
+  });
+
+  if (!resolvedRuntimeVersion?.fingerprintSources) {
+    return {
```

Comment:
nit: add comment to explain the case here is when the `runtimeVersion.policy` is not fingerprint.